xml/Includes.xml

```diff
@@ -1415,7 +1415,6 @@
 			<align>center</align>
 			<aligny>center</aligny>
 			<onclick>Action(Back)</onclick>
-			<visible>Skin.HasSetting(touchmode) + [!Window.IsActive(home) | $EXP[infodialog_active]]</visible>
```

Comment:
No. I've changed the vis conditions for the individual info dialogs like you suggested. And also added the IsActive(home) to the vis condition on the top bar.
So it isn't necessary to make that check anymore.